setup.cfg

```diff
@@ -9,20 +9,16 @@ upload = upload upload_docs --upload-dir build/sphinx/html
 [bdist_rpm]
 doc-files = doc
 
-[nosetests]
-verbosity = 2
-detailed-errors = 1
-with-coverage = 1
-cover-package = joblib
-#pdb = 1
-#pdb-failures = 1
-with-doctest=1
-doctest-extension=rst
-doctest-fixtures=_fixture
-
 [wheel]
 universal=1
 
+[pytest]
+addopts =
+    --cov="joblib"
+    --disable-pytest-warnings
+    --doctest-glob="*.rst"
+    --doctest-modules
```

Comment:
Is there an equivalent of doctest-fixture? AFAIR this is something that gets executed before the .rst doctest are run.